modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/form21a_controller.rb

```diff
@@ -22,14 +31,33 @@ def submit
 
       attr_reader :parsed_request_body
 
+      def schema
+        # NOTE: This doesn't reject any extra attributes not found in the schema. If
+        # we want that the schema needs to have { "additionalProperties" => false }
+        VetsJsonSchema::SCHEMAS[FORM_ID.upcase]
+      end
+
       # Parses the raw request body as JSON and assigns it to an instance variable.
       # Renders a bad request response if the JSON is invalid.
       def parse_request_body
         @parsed_request_body = JSON.parse(request.raw_post)
       rescue JSON::ParserError
-        Rails.logger.error(
-          "Form21aController: Invalid JSON in request body for user with user_uuid=#{@current_user&.uuid}"
-        )
+        handle_json_error
+      end
+
+      def validate_schema
```

Comment:
I think `validate_schema` has the wrong meaning, and we should go with `validate_form` instead.